app/assets/javascripts/discourse/app/models/topic-tracking-state.js

```diff
@@ -1,26 +1,36 @@
 import EmberObject, { get } from "@ember/object";
 import discourseComputed, { on } from "discourse-common/utils/decorators";
 import Category from "discourse/models/category";
+import { deepEqual, deepMerge } from "discourse-common/lib/object";
 import DiscourseURL from "discourse/lib/url";
 import { NotificationLevels } from "discourse/lib/notification-levels";
 import PreloadStore from "discourse/lib/preload-store";
 import User from "discourse/models/user";
-import { deepEqual } from "discourse-common/lib/object";
 import { isEmpty } from "@ember/utils";
 
-function isNew(topic) {
+function isNew(topic, currentUser) {
+  let createdInNewPeriod = true;
+  if (currentUser) {
```

Comment:
I had to add this in because we are now possibly returning topics over the new topic threshold that shouldn't show as "new" in tracking but which can be used for total counts.